now-build.sh

```diff
@@ -14,5 +14,4 @@ export NODE_ENV="production"
 yarn concurrently 'yarn workspace @magento/pwa-buildpack run build' 'yarn workspace @magento/peregrine run build:esm'
 # For the Venia build, any environment variables not set in now.json or other
 # scripts are filled in by copying the default .env file here.
-cp packages/venia-concept/.env.dist packages/venia-concept/.env
```

Comment:
I remember that it was much faster to just copy the package.json files and do an NPM install, because then the installed packages are cached based on the package.json content alone. And then you can revert to the pre-installed stage before with Docker to speed up your install process, as long as you haven't changed the package files or yarn.lock.

TBH it would be easier if Dockerfiles let us copy globs.

Reply:
sorry i missed your comment @jimbo , i don't know if I am missing something, but this is the now-build.sh file, this shouldn't have anything to do with the setup in the Dockerfile.

you might be remembering [this section](https://github.com/magento-research/pwa-studio/blob/develop/Dockerfile.dev#L16-L28) in the Dockerfile where we copy over the dependency files for install and then copy over the rest of the package file that won't change as much to take advantage of Docker cache.